cassandra_nodetool/setup.py

```diff
@@ -1,8 +1,9 @@
 # (C) Datadog, Inc. 2018
```

Comment:
You don't have to touch this file as @zippolyte said in ref to https://github.com/DataDog/integrations-core/pull/1796/files.